saleor/graphql/translations/types.py

```diff
@@ -380,13 +339,6 @@ class Meta:
 
 class SaleTranslatableContent(CountableDjangoObjectType):
     translation = TranslationField(SaleTranslation, type_name="sale")
-    sale = graphene.Field(
```

Comment:
Don't you need to remove resolver for this? I see `resolve_sale`, same for Voucher